assets/src/components/SelectionTool.js

```diff
@@ -117,12 +210,38 @@ export default class SelectionTool extends HTMLElement {
 
         mainEventDispatcher.addListener(
             () => {
-                render(mainTemplate(), this);
+                render(
+                    mainTemplate(
+                        this.isFilterDisabled,
+                        this.results,
+                        mainLizmap.selectionTool.newAddRemoveSelected,
+                        mainLizmap.selectionTool.filteredFeaturesCount
+                    ),
+                    this
+                );
             },
-            ['selectionTool.newAddRemoveSelected', 'selectionTool.allFeatureTypeSelected', 'selection.changed', 'filteredFeatures.changed', 'selection.bufferValue']
+            [
+                'selectionTool.newAddRemoveSelected',
+                'selectionTool.allFeatureTypeSelected',
+                'selection.changed',
+                'filteredFeatures.changed',
+                'selection.bufferValue'
```

Comment:
```suggestion
                'selection.bufferValue',
```

Neat-picking, but having a trailing comma is better, it avoids the next user to add one if he wants to add a line at the end of the list, and to be have the line `selection.bufferValue` updated, while he didn't change anything to this line. I would agree to set this as a rule for future contributions.

While you are here, I would make it alphabetic, but that's also my own opinion. There is a usually a control for that in an IDE.